tools/pyclient/README.md

```diff
@@ -39,6 +44,32 @@ with Client('https://example.molgeniscloud.org') as client:
     # Delete a schema from the server
     client.delete_schema(name='New Schema')
 
+```
+Instead of signing in with a username and password the client can also be used while authorized by a (temporary) token that is generated on the server.

```

Comment:
Link back 'token' to use_tokens.md

Reply:
I tried doing this, but there's not an "independent" page I can link to

Reply:
Fixed this!